README.md

```diff
@@ -1 +1,10 @@
 # live-fanatic-backend
+Needed dependencies 
+
+npm i (both in live-only-fanatic & in live-only-fanatic/app) 
+npm i -g concurrenly
+npm i -g nodemon
+
+- for search bar component -
+npm install @material-ui/icons
```

Comment:
is this needed every time? aren't these already set as dependencies and get added with npm i?

Reply:
möjligtvis, har inte provat hitintills med bara npm i